src/pQueue.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+export default class pQueue {
+  constructor() {
+    this.storage = []
+    this.top = 0
+  }
+
+  enqueue(name, priority) {
+    this.storage[this.top++] = {name, priority}
+    return this.storage
```

Comment:
`enqueue` should not return anything.  As well, this is a particularly risky implementation as it exposes the internal storage structure to clients of the `PriorityQueue`.  Our intent in creating objects is to encapsulate or hide the internal workings of the object, and only allow changes through the publicly defined API (like `enqueue`, `dequeue`, `isEmpty`, etc.)